en.md

```diff
@@ -56,7 +56,52 @@ We set one or several persistent cookies in order to save the user’s settings
 Persistent cookies are automatically deleted after a specified length of time, which may differ from one cookie to another. You may delete the cookies in the security settings of your browser at any time.
 You may configure your browser setting the way you want it and e.g. refuse the acceptance of third-party cookies or all cookies. Cookies that have already been saved may be deleted at any time. This can also be done automatically. Please note that you may not be able to use all functions of this website within the full scope if you deactivate cookies for our website.
 
-## 6. Special forms of use of the Ecosia website
+## 6. Mobile Apps
+
+The Ecosia app provides another means of access to the Ecosia platform. Accordingly, we use your data within the app as if you were accessing Ecosia via our website. This applies, for example, also to the use of your data and cookies, which will be used for the same purposes on the same legal basis. Thus, the statements made in the present privacy policy will apply to the app.
+
+### a. Additional use of app-specific data
+
+We use a safe and fully encrypted analysis service, which we ourselves operate, in order to analyse and continuously improve the use of our app. Using the statistics obtained, we can improve our offering and design it in a more interesting way for you as a user. The legal basis for this processing of your personal data is Art. 6 Abs. 1 lit. f of the General Data Protection Regulation.
+
+Additionally to the data described under 7.g. we collect the following data when you use our app:
+
+ - Usage statistics of the app, e.g. which interface elements are interacted with
+ - Data relating to your app or your end device, e.g. the name of the device, its manufacturer, model, operating system number, app number or SDK version and the device number (IMEI)
+
+The Ecosia app may use your locational data in anonymized, respectively pseudonymized, form if you have allowed access to these data on your device. We use the locational data in order to give you a better Ecosia experience or to show more relevant search results or advertising to you. Personal data such as names, addresses or dates of birth will not be collected, stored or transmitted.
+
+These data are anonymised after a short space of time, generally 7 days. The anonymised data are saved for as long as is necessary for the evaluation. The data are not transferred to any third party. We will not access your data located outside of the app, such as your calendar, photos, messages or such like unless you have permitted us to do so.
```

Comment:
The word "data" is only singular in this context, so this should read "This data is anonymised".

Reply:
Also, in the second sentence: "The anonymised data is saved".
Third sentence: "The data is not transferred"

Reply:
@jasongwartz Argh thats what it read first and then Trudie changed it into plural when proofreading it. Well at least its just a grammar change no content change that needs to be reflected in the translations.

Reply:
Ok. I can talk to Trudie about it, but I am 10000% sure.

Reply:
@jasongwartz No need it also sounds better to me. Changed it.